src/main/multiparser.js

```diff
@@ -91,10 +81,36 @@ async function printEmbeddedLanguages(
       }
     }
 
-    const embeddedLanguage = detectEmbeddedLanguage(path, options);
-    if (embeddedLanguage) {
-      pathStacks.push({ pathStack: [...path.stack], embeddedLanguage });
+    let result;
+    try {
```

Comment:
Do we need this try-catch? I think you added it to preserve compatibility with the current behavior of `embed`, but we're fully changing this behavior anyway. The main (and probably only) purpose of the try-catch around `embed` is to catch parser errors.

Reply:
Make sense to remove it.